app/services/job_tracker.rb

```diff
@@ -0,0 +1,45 @@
+class JobTracker
+  VALID_STATE_KEYS = [:message, :state].freeze
+  DAY_IN_SECONDS = 86_400
```

Comment:
`1.day.seconds` ?

If that works, you can remove the constant and use directly in L13